runtime/src/configs/mod.rs

```diff
@@ -309,32 +309,35 @@ impl pallet_collator_selection::Config for Runtime {
 parameter_types! {
     // Storage Provider Pallet
     pub const WpostProvingPeriod: BlockNumber = DAYS;
-    // Half an hour (=48 per day)
-    // 30 * 60 = 30 minutes
-    // SLOT_DURATION is in milliseconds thats why we / 1000
-    pub const WpostChallengeWindow: BlockNumber = 30 * 60 / (SLOT_DURATION as BlockNumber / 1000);
-    pub const WPoStChallengeLookBack: BlockNumber = 10 * MINUTES;
-    pub const MinSectorExpiration: BlockNumber = 180 * DAYS;
-    pub const MaxSectorExpiration: BlockNumber = 1278 * DAYS;
-    pub const SectorMaximumLifetime: BlockNumber = (365 * DAYS) * 5; // 5 years
-    pub const MaxProveCommitDuration: BlockNumber =  (30 * DAYS) + 150;
+    // There are 48 deadlines in a day, so a deadline is 30 minutes.
     pub const WPoStPeriodDeadlines: u64 = 48;
-    pub const MaxPartitionsPerDeadline: u64 = 3000;
-    pub const DeclarationsMax: u64 = 3000;
-    pub const FaultMaxAge: BlockNumber = DAYS * 42;
-    /// Copied from FileCoin and adapted to substrate block time. WPoStChallengeLookBack + 125 blocks
-    /// <https://github.com/filecoin-project/builtin-actors/blob/6906288334746318385cfd53edd7ea33ef03919f/runtime/src/runtime/policy.rs#L327>
-    pub const FaultDeclarationCutoff: BlockNumber = (10 * MINUTES) + 125;
-    /// <https://github.com/filecoin-project/builtin-actors/blob/a45fb87910bca74d62215b0d58ed90cf78b6c8ff/runtime/src/runtime/policy.rs#L306>
-    pub const PreCommitChallengeDelay: BlockNumber = 75 * MINUTES;
+    pub const WpostChallengeWindow: BlockNumber = 30 * MINUTES;
+    // PoSt for a single partition is ~20sec, it's a huge margin to start generating the proofs before deadline starts.
+    pub const WPoStChallengeLookBack: BlockNumber = 10 * MINUTES;
+
+    // Finality in polkadot is 12-60 seconds, gives us a finalized randomness to base the challenge on.
+    pub const PreCommitChallengeDelay: BlockNumber = 1 * MINUTES;
+    // With 1GiB sector size and max security params it takes ~22 minutes to create a replica and zk-SNARK on top of it.
+    pub const MaxProveCommitDuration: BlockNumber =  30 * MINUTES;
+    // The chain can verify 1 proof per block (of 25 sectors per partitions) currently.
+    // It means deadline time (blocks) == max theoretical number of the partitions per deadline.
+    // There is no global limit, so currently 1 Storage Provider could overload the entire chain.
+    // 1 deadline = 30 minutes, 30 * 10 blocks = 300.
+    pub const MaxPartitionsPerDeadline: u64 = 300;
+
+    // Fault declaration/recovery functionality has been postponed, this won't make a difference until #592 is done.
+    pub const FaultMaxAge: BlockNumber = 1 * DAYS;
+    /// WPoStChallengeLookBack + a tiny margin.
+    pub const FaultDeclarationCutoff: BlockNumber = (10 * MINUTES) + (1 * MINUTES);
```

Comment:
Maybe just use the value then?
```suggestion
    /// WPoStChallengeLookBack + a tiny margin.
    pub const FaultDeclarationCutoff: BlockNumber = WPoStChallengeLookback + (1 * MINUTES);
```